core/router.js

```diff
@@ -17,7 +17,7 @@
 var bloomrun = require('bloomrun')
 var assert = require('assert')
 var stringify = require('fast-safe-stringify')
-var errors = require('./err')
+var mue = require('mu-error')({dev: process.NODE_ENV !== 'production'})
```

Comment:
I don't think this should happen here, but it should be a configuration of `mu` itself